src/AspNet.Security.OAuth.Apple/Internal/DefaultAppleIdTokenValidator.cs

```diff
@@ -18,24 +18,21 @@ internal sealed class DefaultAppleIdTokenValidator : AppleIdTokenValidator
     {
         private readonly ILogger _logger;
         private readonly AppleKeyStore _keyStore;
-        private readonly JwtSecurityTokenHandler _tokenHandler;
         private readonly CryptoProviderFactory _cryptoProviderFactory;
 
         public DefaultAppleIdTokenValidator(
             [NotNull] AppleKeyStore keyStore,
-            [NotNull] JwtSecurityTokenHandler tokenHandler,
             [NotNull] CryptoProviderFactory cryptoProviderFactory,
             [NotNull] ILogger<DefaultAppleIdTokenValidator> logger)
         {
             _keyStore = keyStore;
-            _tokenHandler = tokenHandler;
             _cryptoProviderFactory = cryptoProviderFactory;
             _logger = logger;
         }
 
         public override async Task ValidateAsync([NotNull] AppleValidateIdTokenContext context)
         {
-            if (!_tokenHandler.CanValidateToken)
+            if (!context.Options.JwtSecurityTokenHandler!.CanValidateToken)
```

Comment:
Is the null-forgiving operator necessary here?

Reply:
Won't compile without it because we're no longer checking for null.

Reply:
But the `JwtSecurityTokenHandler` is not "nullable", right? It shouldn't be needed here (if it is, it may be another Roslyn bug).

Reply:
Looks like that bug was fixed in RC2.